cmd/zetae2e/config/localnet.yml

```diff
@@ -0,0 +1,55 @@
+zeta_chain_id: "athens_101-1"
+accounts:
```

Comment:
To me it seems we're adding configuration opiniated to local tests. So far the config was fully configured whether you use for local or live networks.

I think we should either:
- Make the account list more generic like having a `additional_accounts` array `{name, address, private_key}`
- Putting this account in a explicit field like `localnet_accounts` to make it clear these doesn't need to be initialized for live network tests

Reply:
I think I prefer the first option as some of these tests could be adjusted to run on live networks in the future. I think this should be a general goal with all e2e tests.

> Make the account list more generic like having

That was my original approach in d2cb7a36130318ad5327fae33da209588dea0a95 but the loss of type safety did not feel quite right. Please advise if you prefer that approach.

Reply:
> I prefer a combination of both options where we use a strongly typed structure but move the optional accounts into a separate additional_accounts field.

Sounds good, I was thinking about using `localnet_accounts` but if we can consider using those in live networks in the future, `additional_accounts ` works